app/controllers/application_controller.rb

```diff
@@ -4,7 +4,7 @@
 
 class ApplicationController < ActionController::Base
   include CurrentUser
-  include CanonicalURL::ControllerExtensions
+  include CanonicalUrl::ControllerExtensions
```

Comment:
Is it possible to keep the constant as `CanonicalURL`? If I remember correctly, I think we can do so with Rails's inflector API.

Reply:
Yes we can and it was done like that in a previous version of this PR but we had some discussions about it [here](https://github.com/discourse/discourse/pull/15649#discussion_r807985251) and [here](https://github.com/discourse/discourse/pull/15649#discussion_r809852869)

Reply:
I believe we can fix this by adding URL as an acronym?